x/leverage/keeper/token.go

```diff
@@ -58,22 +58,14 @@ func (k Keeper) SetRegisteredToken(ctx sdk.Context, token types.Token) {
 		panic(fmt.Sprintf("failed to encode token: %s", err))
 	}
 
-	// for tokens not previously registered, set tokens:uToken to 1.0
-	if err := k.InitializeExchangeRate(ctx, token.BaseDenom); err != nil {
```

Comment:
Utoken exchange rates are derived instantly, so initializing is no longer required